pages/index.js

```diff
@@ -118,7 +118,7 @@ export default function Home() {
               <p data-testid='landing-page-gas-warning'> If you suspect you have a gas leak, you must report it immediately to the Gas Emergency Service on <strong>0800 111 999</strong> or via textphone (minicom) on <strong>0800 371 787</strong> </p>
               <h2 className={'govuk-heading-m'}>Report a repair</h2>
               <p data-testid='landing-page-report-repair-text'>
-                Report a repair for your council property including leasehold or communal areas
+                Report a repair for your council property {enableLeaseHolderFlow() ? 'including leasehold or communal areas' : 'including communal areas'}
```

Comment:
I believe this line could be simplified - see suggestion
```suggestion
                Report a repair for your council property including {enableLeaseHolderFlow() ? 'leasehold or ' : ''}communal areas
```